pyjexl/operators.py

```diff
@@ -2,12 +2,25 @@
 
 
 class Operator(object):
-    __slots__ = ('symbol', 'precedence', 'evaluate')
+    __slots__ = ('symbol', 'precedence', 'evaluate', '_evaluate_lazy')
 
-    def __init__(self, symbol, precedence, evaluate):
+    def __init__(self, symbol, precedence, evaluate, evaluate_lazy=False):
+        """Operator definition.
+
+        If evaluate_lazy is se to True, the `evaluate()` method will receive
```

Comment:
```suggestion
        If evaluate_lazy is set to True, the `evaluate()` method will receive
```

Reply:
I was hoping I could easily do this myself. Can you apply this change, either through the Github UI or by amending your original commit? Then this should be ready to merge.

Reply:
Sure! Fixed